app/views/helpers/ServicePageHelper.scala

```diff
@@ -103,8 +104,14 @@ object ServicePageHelper {
     val currentReturnPeriod = ReturnPeriod(LocalDate.now)
     val lastPeriodStart = currentReturnPeriod.previous.start.format(monthFormatter)
     val lastPeriodEnd = currentReturnPeriod.previous.end.format(monthYearFormatter)
-    val submittedTime = lastReturn.submittedOn.getOrElse(Instant.now).atZone(ZoneId.of("Europe/London")).format(timeFormatter).toLowerCase
-    val submittedDate = lastReturn.submittedOn.getOrElse(Instant.now).atZone(ZoneId.of("Europe/London")).format(dateFormatter)
+    val submittedTime = lastReturn.submittedOn
```

Comment:
why we should define submitted time here again we have another one in line 81

Reply:
we have two separte functions noReturnsPendingMessage, finalReturnSentMessage. I'll make it more concise